smctemp.cc

```diff
@@ -485,5 +485,39 @@ double SmcTemp::GetCpuTemp() {
   return temp;
 }
 
+double SmcTemp::GetGpuTemp() {
+  double temp = 0.0;
+#if defined(ARCH_TYPE_X86_64)
+  temp = smc_accessor_.ReadValue(kSensorTg0d);
+  if (0.0 < temp && temp < 110.0) {
```

Comment:
How about creating new private method `bool IsValidTemp(double)`?

Reply:
I think it's a good idea. Added it on a new commit since ideally we should reuse the method also in the CPU code